src/libraries/System.Private.CoreLib/src/System/Threading/NativeRuntimeEventSource.PortableThreadPool.cs

```diff
@@ -8,11 +8,52 @@
 
 namespace System.Diagnostics.Tracing
 {
-    // This is part of the NativeRuntimeEventsource for Mono, which is the managed version of the Microsoft-Windows-DotNETRuntime provider.
-    // and contains the implementation of ThreadPool events.
+    // This is part of the NativeRuntimeEventsource for Mono|CoreRT, which is the managed version of the Microsoft-Windows-DotNETRuntime provider.
+    // and contains the implementation of ThreadPool events. Since file is shared between Mono|CoreRT it is kept in shared part of SPC.
     // To look at CoreCLR implementation of these events, refer to NativeRuntimeEventSource.PortableThreadPool.CoreCLR.cs.
     internal sealed partial class NativeRuntimeEventSource : EventSource
     {
+        // On Mono|CoreRT, we don't have these keywords defined from the genRuntimeEventSources.py, so we need to manually define them here.
+        public class Keywords
+        {
```

Comment:
This file is specific to PortableThreadpool. We should only include the PortableThreadpool-specific keywords here.

(Or undo this change and keep the list under ifdef like it is now.)

Reply:
Tried to get away from ifdef:ing these things when moving into runtime specific SPC source files. Do you need these keywords on CoreRT? If not, we could moved them into NativeRuntimeEventSource.PortableThreadPool.Mono.cs shim file closely matching the generated file on CoreClr.

Reply:
Looks like most of them are only used in NativeRuntimeEventSource.CoreCLR.cs, except the Threading keyword, so maybe we can exclude all except that one for now.

Reply:
Yes, that works.